pkg/v1/providers/ytt/vendir/cni/_ytt_lib/addons/packages/antrea/1.5.2/bundle/config/schema.yaml

```diff
@@ -0,0 +1,96 @@
+    tunnelType: none
+    #@schema/desc "Determines how tunnel traffic is encrypted. One of the following options => none, ipsec, wireGuard"
+    trafficEncryptionMode: none
+    #@schema/desc "WireGuard related configurations."
+    wireGuard:
+      #@schema/desc "The port for WireGuard to receive traffic."
+      port: 51820
+    #@schema/desc "Enable usage reporting (telemetry) to VMware."
+    enableUsageReporting: false
+    #@schema/desc "ClusterIP CIDR range for IPv4 Services"
+    serviceCIDR: 10.96.0.0/12
+    #@schema/desc "ClusterIP CIDR range for IPv6 Services"
+    #@schema/nullable
+    serviceCIDRv6: ""
+    #@schema/desc "The traffic encapsulation mode. One of the following options => encap, noEncap, hybrid, networkPolicyOnly"
+    trafficEncapMode: encap
+    #@schema/desc "Flag to enable/disable SNAT for the egress traffic from a Pod to the external network"
+    noSNAT: false
+    #@schema/desc "Disable UDP tunnel offload feature on default NIC"
+    disableUdpTunnelOffload: false
+    #@schema/desc "Default MTU to use for the host gateway interface and the network interface of each Pod"
+    #@schema/nullable
+    defaultMTU: ""
+    #@schema/desc "List of allowed cipher suites. If omitted, the default Go Cipher Suites will be used"
+    tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
+    #@schema/desc "FeatureGates is a map of feature names to flags that enable or disable experimental features"
+    featureGates:
+      #@schema/desc "Flag to enable/disable antrea proxy"
+      AntreaProxy: true
+      #@schema/desc "Flag to enable/disable EndpointSlice support in AntreaProxy. If AntreaProxy is not enabled, this flag will not take effect"
+      EndpointSlice: false
+      #@schema/desc "Flag to enable/disable antrea traceflow"
+      AntreaTraceflow: true
+      #@schema/desc "Flag to enable/disable NodePortLocal feature to make the pods reachable externally through NodePort"
+      NodePortLocal: true
+      #@schema/desc "Flag to enable/disable antrea policy"
+      AntreaPolicy: true
+      #@schema/desc "Flag to enable/disable flow exporter"
+      FlowExporter: false
+      #@schema/desc "Flag to enable/disable network policy stats"
+      NetworkPolicyStats: false
+      #@schema/desc "Flag to enable/disable SNAT IPs of Pod egress traffic"
+      Egress: true
+      #@schema/desc "Flag to enable/disable node IPAM controller"
+      NodeIPAM: false
+      #@schema/desc "Flag to enable/disable flexible IPAM mode"
+      AntreaIPAM: false
+      #@schema/desc "Flag to enable/disable managing external IPs for Load balancers services"
+      ServiceExternalIP: false
+      #@schema/desc "Flag to enable/disable multicast traffic"
+      Multicast: false
```

Comment:
To use this commit as a reference, we need to add images info to schemas.
https://github.com/vmware-tanzu/community-edition/commit/aa3bcff00998b57a8c93738f98b32ad6f2d18f5d
Otherwise the test won't pass because of the image info are blocked by schemas.
https://github.com/vmware-tanzu/tanzu-framework/blob/f4b11f0906bfca86cb82c3f2545631af8da73bb9/pkg/v1/providers/ytt/02_addons/cni/antrea/antrea_addon_data.lib.yaml#L70-L74

Reply:
And also consider to remove this image info in data values.yaml from now on. Since it is a deprecated field because it is useless for packages.